eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1896,6 +1930,13 @@ class Channel(val nodeParams: NodeParams, wallet: EclairWallet, remoteNodeId: Pu
       case _ => throw CommitmentSyncError(d.channelId)
     }
 
+    commitments1.remoteNextCommitInfo match {
+      case Left(_) =>
+        // we expect a quick response from our peer
```

Comment:
It's the same comment as when we've just sent a signature but it's a different context and it should be different here